openassessment/staffgrader/tests/test_submission_lock_model.py

```diff
@@ -0,0 +1,74 @@
+from datetime import datetime, timedelta, timezone
+from openassessment.staffgrader.errors.submission_lock import SubmissionLockContestedError
+
+from openassessment.staffgrader.models.submission_lock import SubmissionGradingLock
+
+from django.test import TestCase
+from freezegun import freeze_time
+
+
+@freeze_time("1969-07-21 02:56:00", tz_offset=0)
+class TestSubmissionLockModel(TestCase):
+    """ Tests for interacting with submission grading/locking """
+    existing_submission_lock = None
+
+    test_submission_uuid = "currently_locked"
+    test_user_id_1 = "foo"
+    test_user_id_2 = "bar"
+    test_timestamp = "1969-07-20T22:56:00-04:00"
+
+    test_submission_uuid_without_lock = "not_currently_locked"
+
+    def setUp(self):
+        self.existing_submission_lock = SubmissionGradingLock.objects.create(
+            submission_uuid=self.test_submission_uuid,
+            owner_id=self.test_user_id_1,
+            created_at=datetime.now(tz=timezone.utc)
+        )
+
+    def test_lock_active(self):
+        # A lock created within the TIMEOUT is considered active
+        assert self.existing_submission_lock.is_active is True
+
+    def test_lock_inactive(self):
+        # A lock created and left for more than TIMEOUT is considered inactive
+        self.existing_submission_lock.created_at = self.existing_submission_lock.created_at - timedelta(hours=12)
```

Comment:
why not use `TIMEOUT` variable?

Reply:
Valid, I just needed "time in the future past timeout" but I can make that clearer